server/capabilities/gke_capabilities.go

```diff
@@ -4,21 +4,26 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"github.com/rancher/kontainer-engine/drivers/gke"
-	"github.com/sirupsen/logrus"
-	"golang.org/x/oauth2"
-	"golang.org/x/oauth2/google"
-	"google.golang.org/api/container/v1"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"strings"
+
+	"github.com/rancher/kontainer-engine/drivers/gke"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	computev1 "google.golang.org/api/compute/v1"
+	containerv1 "google.golang.org/api/container/v1"
+	iamv1 "google.golang.org/api/iam/v1"
```

Comment:
Can we let these package names default? I don't generally like overriding the default names unless there is a collision.  _Especially_ so if we are indicating a version in our package names.  That may be confusing down the line if we update to v2 and someone forgets to update the name.